src/view/UserphraseView.cpp

```diff
@@ -53,9 +54,12 @@ void UserphraseView::showModifyUserphraseDialog()
     dialogType_ = DIALOG_MODIFY;
 
     auto userphrase = model()->getUserphrase(selectionModel()->selectedIndexes().first());
-
     UserphraseDialog_->setText(userphrase->phrase_, userphrase->bopomofo_);
     UserphraseDialog_->setWindowTitle(tr("Modify phrase"));
+    
+    *originalPhrase = UserphraseDialog_->getPhrase();
+    *originalBopomofo = UserphraseDialog_->getBopomofo();
```

Comment:
Ok, I will use `userphrase->phrase_` and `userphrase->bopomofo_` to replace `UserphraseDialog_->getPhrase()`, `UserphraseDialog_->getBopomofo()` (I misthought before)

I have another question: should I replace the parameter of `UserphraseDialog_->setText(userphrase->phrase_, userphrase->bopomofo_)` to UserphraseDialog_->setText(*originalPhrase, *originalBopomofo)?